src/app/models/form.server.model.ts

```diff
@@ -577,6 +577,24 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     )
   }
 
+  // Deletes specified form logic.
+  FormSchema.statics.deleteFormLogic = async function (
```

Comment:
can we add unit tests for this also pls?

Reply:
added